client/src/components/ProfileMenu.js

```diff
@@ -25,11 +28,17 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const ProfileMenu = () => {
+const ProfileMenu = (props) => {
   const [anchorEl, setAnchorEl] = useState(null);
   const classes = useStyles();
+  const value=useContext(UserContext)
```

Comment:
Missing semi colons here and below. Can you check that your linter is set to format on save? 
(I just had to fix mine today in VS Code, so there might have been an issue with the Prettier extension)